<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataItem</class>
 <widget class="QWidget" name="DataItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataItem</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="btnImage">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnMarkDel">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lnData"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
